Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -105,7 +105,7 @@ inline void ComputeStrokeStyle(ID2D1DeviceContext* deviceContext) {
         deviceContext->GetFactory(&factory);
 
         std::vector<float> adjustedDashes(dashes.size());
-        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const float& f) -> float { return f / lineWidth; });
+        std::transform(dashes.cbegin(), dashes.cend(), adjustedDashes.begin(), [this](const CGFloat& f) -> float { return f / lineWidth; });
```

Comment:
thought I added a different comment here... try [lineWidth = this.lineWidth] or something to that effect.  and is -> float necessary?